cuda_core/cuda/core/experimental/_launcher.py

```diff
@@ -146,6 +146,6 @@ def launch(stream, config, kernel, *kernel_args):
         # TODO: check if config has any unsupported attrs
         handle_return(
             driver.cuLaunchKernel(
-                int(kernel._handle), *config.grid, *config.block, config.shmem_size, stream._handle, args_ptr, 0
+                int(kernel._handle), *config.grid, *config.block, config.shmem_size, stream._ctx_handle, args_ptr, 0
```

Comment:
We need a stream here, not context:
```suggestion
                int(kernel._handle), *config.grid, *config.block, config.shmem_size, stream.handle, args_ptr, 0
```